### ISSUE TITLE
Accessibility: My Site - Create Button and Tooltip Issue

### ISSUE THREAD

**Issue by jd-alexander** [wordpress-mobile/WordPress-Android#11161]
### Expected behavior
1. Create FAB should say "create a post or page" when TalkBack selects it.  
2. The tooltip should be inaccessible or even disabled in accessibility mode since it blocks the "Site Pages" button under publishing. 

### Actual behavior

1. Create FAB says "Create a post" even though it creates posts and pages. 
<img src="https://user-images.githubusercontent.com/1509205/73106246-f56a7f00-3ec8-11ea-828f-e1fc8b3563d1.png" width="320">

2. The tooltip is activated and it says tap to activate even though that does nothing. 
<img src="https://user-images.githubusercontent.com/1509205/73106378-45e1dc80-3ec9-11ea-940e-ebac4770efc7.png" width="320">

### Steps to reproduce the behavior
1. Enable TalkBack. 
2. Open the app and ensure you are on the `My Site` tab. 
3. Select the FAB and tooltip. 

Tested on Google Pixel 4 XL, Android 10, WPAndroid `develop` ()
